frontend/src/components/BalancePresentation.js

```diff
@@ -50,7 +49,7 @@ class BalancePresentation extends React.Component {
             <p>{balanceLabel}</p>
           </Col>
           <Col className="pl-0 d-flex justify-content-end">
-            <p>{this.state.balance} €</p>
+            <p>{this.props.balance} €</p>
```

Comment:
Rather than taking balance in as a prop, I feel it would be more simple just to calculate it here based on the other props/state values. This way we would not need any change in the LandingMain component for this calculation. E.g.
```jsx
<p>{this.props.salary - this.state.saving - this.state.expenses} €</p>
```

Reply:
done